src/components/apiTokens/CreateAPIToken.tsx

```diff
@@ -208,20 +236,21 @@ function CreateAPIToken({
     }
 
     return (
-        <>
-            <div className="cn-9 fw-6 fs-16">
-                <span className="cb-5 cursor" onClick={redirectToTokenList}>
-                    API tokens
-                </span>{' '}
-                / New API token
+        <div className='bcn-0 api__token'>
+            <div className='flex dc__content-space pt-16 pb-16 dc__gap-8'>
+                <div className='flex row ml-0 h-32'>
+                    <div className="cn-9 fw-6 fs-16">
+                        <span className="cb-5 cursor" onClick={redirectToTokenList}>
+                            {API_COMPONENTS.TITLE}
+                        </span>{' '}
+                        {API_COMPONENTS.NEW_API_TITLE}
+                    </div>
+                    {handleQuestion()}
```

Comment:
Please rename it to renderQuestionwithTippy or something better

Reply:
done